templates/skeleton/server.ts

```diff
@@ -70,7 +82,14 @@ export default {
       const handleRequest = createRequestHandler({
         build: remixBuild,
         mode: process.env.NODE_ENV,
-        getLoadContext: () => ({session, storefront, cart, env, waitUntil}),
+        getLoadContext: () => ({
+          session,
+          storefront,
+          customerClient,
```

Comment:
Considering we call the Storefront client `storefront`, should we call this `customer` or `customerAccount`? 🤔

Reply:
I honestly debate over the two a lot 😂  
the offical name for the api is [`Customer Account API`](https://shopify.dev/docs/api/customer) so maybe everything should be named `customerAccount` or `customer-account` accordingly?

Reply:
No strong opinions. `customer` just seemed need and terse. `customerAccount` might work as well.